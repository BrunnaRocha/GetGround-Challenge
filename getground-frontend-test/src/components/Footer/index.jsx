import React from "react";
import styled from "styled-components";

const Container = styled.footer`
  display: flex;
  justify-content: center;
  align-items: center;
  width: 100%;
  height: 50px;

  strong {
    color: #1C444E;
    font-weight: 700;
    text-align: center;

    & a {
      color: #E26227;
    }
  }
`;


export default function Footer() {
  return (
    <Container>
      <strong>
        Developed by{" "}
        <a
          href="https://github.com/BrunnaRocha"
          target="_blank"
          rel="noreferrer"
        >
          Brunna Rocha
        </a>
      </strong>
    </Container>
  );
}