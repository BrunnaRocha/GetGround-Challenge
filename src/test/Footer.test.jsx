import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../components/Footer'

function toJson(component) {
  const result = component.toJSON()
  expect(result).toBeDefined()
  expect(result).not.toBeInstanceOf(Array)
  return result
}

test('Footer match snapshot', () => {
  const component = renderer.create(
    <Footer />,
  )
  let tree = toJson(component)
  expect(tree).toMatchSnapshot()
})