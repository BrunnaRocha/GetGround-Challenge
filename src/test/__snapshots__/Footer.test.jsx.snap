// Vitest Snapshot v1

exports[`Footer match snapshot 1`] = `
<footer
  className="sc-bdvvtL iNBamC"
>
  <strong>
    Developed by
     
    <a
      href="https://github.com/BrunnaRocha"
      rel="noreferrer"
      target="_blank"
    >
      Brunna Rocha
    </a>
  </strong>
</footer>
`;
